api/v1alpha1/experiment_types.go

```diff
@@ -88,6 +88,10 @@ type Metric struct {
 	Name string `json:"name"`
 	// Indicator that the goal of the experiment is to minimize the value of this metric
 	Minimize bool `json:"minimize,omitempty"`
+	// The inclusive minimum allowed value for the metric
+	Min *resource.Quantity `json:"min,omitempty"`
```

Comment:
Do we really want to use `resource.Quantity` for this instead of a float ( since that is what metrics evaluate to)?
Slightly concerned about people putting in something like `100m`

Reply:
We can't use floats in CRDs, this was the best compromise. Nothing stops people from specifying `100m` as a bound. I did crank up the scale of the quantity though, by using nano scale instead of milli scale you can specify what appears to be more precise decimal numbers.